app/models/post.rb

```diff
@@ -182,7 +182,12 @@ def expired?
   end
 
   def parsed_controls
-    JSON.parse(self.controls)
+    begin
+      JSON.parse(self.controls).presence || []
+    rescue JSON::ParserError
+      self.update controls: "[]"
```

Comment:
This seems a little dangerous, but at the same time I kind of like it. I do feel like it needs `.save(touch: false)` so as to not update the `updated_at` column.

Reply:
Good call on `touch: false`, I'll go ahead and add that. As for the manual update, this wouldn't erase any data that was meaningful (ideally), since only failed JSON parses would fall back to updating the model itself. If we notice people complaining of controls not being saved, we'll have to retackle #71.